src/Hyperwallet.js

```diff
@@ -1799,4 +1799,181 @@ export default class Hyperwallet {
+     *
+     * @param {string} userToken - The user token
+     * @param {string} stakeholderToken - The user token
+     * @param {Object} data - The Stakeholder data that should be updated
+     * @param {api-callback} callback - The callback for this call
+     *
+     * @throws Will throw an error if userToken is not provided
+     */
+    updateBusinessStakeholder(userToken, stakeholderToken, data, callback) {
+        if (!userToken) {
+            throw new Error("userToken is required");
+        }
+        if (!stakeholderToken) {
+            throw new Error("stakeholderToken is required");
+        }
+        this.client.doPut(`users/${encodeURIComponent(userToken)}/business-stakeholders/${encodeURIComponent(stakeholderToken)}`, data, {}, callback);
+    }
+
+    /**
+     * List all Business Stakeholder
+     *
+     * @param {string} userToken - The user token
+     * @param {Object} options - The query parameters to send
+     * @param {api-callback} callback - The callback for this call
+     *
+     * @throws Will throw an error if userToken is not provided
+     */
+    listBusinessStakeholder(userToken, options, callback) {
+        if (!userToken) {
+            throw new Error("userToken is required");
+        }
+        this.client.doGet(`users/${encodeURIComponent(userToken)}/business-stakeholders`, options, Hyperwallet.handle204Response(callback));
+    }
+
+    /**
+     * Activate a Business Stakeholder transition
+     *
+     * @param {string} userToken -  user token
+     * @param {api-callback} callback -  callback for this call
+     * @throws Will throw an error if userToken is not provided
+     */
+    activateBusinessStakeholder(userToken, stakeholderToken, callback) {
+        if (!userToken) {
+            throw new Error("userToken is required");
+        }
+        if (!stakeholderToken) {
+            throw new Error("stakeholderToken is required");
+        }
+        const transition = {
+            transition: "ACTIVATED",
+        };
+        this.client.doPost(`users/${encodeURIComponent(userToken)}/business-stakeholders/${encodeURIComponent(stakeholderToken)}/status-transitions`, transition, {}, callback);
+    }
+
+    /**
+     * Deactivate a Business Stakeholder transition
+     *
+     * @param {string} userToken -  user token
+     * @param {api-callback} callback -  callback for this call
+     * @throws Will throw an error if userToken is not provided
+     */
+    deactivateBusinessStakeholder(userToken, stakeholderToken, callback) {
+        if (!userToken) {
+            throw new Error("userToken is required");
+        }
+        if (!stakeholderToken) {
+            throw new Error("stakeholderToken is required");
+        }
+        const transition = {
+            transition: "DE_ACTIVATED",
+        };
+        this.client.doPost(`users/${encodeURIComponent(userToken)}/business-stakeholders/${encodeURIComponent(stakeholderToken)}/status-transitions`, transition, {}, callback);
```

Comment:
Could you change this to call the createBusinessStakeholderStatusTransition, instead of making an API call from here.